pkg/project/version_bundle_aws.go

```diff
@@ -8,9 +8,11 @@ var versionBundleAWS = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
 			Component:   Name(),
-			Description: "TODO",
+			Description: "Enable NodePort ingress service on KVM.",
```

Comment:
should we have KVM changes in AWS version bundle? And s versionBundle even still used?

Reply:
42 :wink: